pollination/honeybee_vtk/export_images.py

```diff
@@ -0,0 +1,79 @@
+from dataclasses import dataclass
+from pollination_dsl.function import Function, command, Inputs, Outputs
+from pydantic.typing import IntStr
+
+
+@dataclass
+class ExportImages(Function):
+    """Export images from am hbjson file."""
+
+    hbjson_file = Inputs.file(
+        description='HBJSON file.',
+        path='input.hbjson'
+    )
+
+    name = Inputs.str(
+        description='Name of image files.',
+        default='Camera'
+    )
+
+    folder = Inputs.folder(
+        description='Target folder where the images will be written.',
+        path='output_folder',
+        default='.'
+    )
+
+    image_type = Inputs.str(
+        description='Choose the type of image file.',
+        default='jpg'
+    )
+
+    image_width = Inputs.int(
+        description='Width of images in pixels.',
+        default=2500
+    )
+
+    image_height = Inputs.int(
+        description='Height of images in pixels.',
+        default=2500
+    )
+
+    background_color = Inputs.str(
+        description='Background color for the images as a string of rgb values.',
+        default='255 255 255'
+    )
+
+    model_display_mode = Inputs.str(
+        description='Set display mode fro the model.',
+        default='shaded'
+    )
+
+    grid_options = Inputs.str(
+        description='Export sensor grids as either points or meshes.',
+        default='ignore'
+    )
+
+    grid_display_mode = Inputs.str(
+        description='Set display mode for the Sensorgrids.',
+        default='surfacewithedges'
+    )
+
+    config_path = Inputs.path(
+        description='File path to the config json file which can be used to mount'
+        'simulation data on HBJSON.',
+        path='config.json'
+    )
```

Comment:
Should we make this an optional input so one can use this command to render images from the building without the results too?

@chriswmackey, this one is fine - right? I forgot which optional artifact has issues (between inputs and outputs). Let us know what's the best way to handle this.

Reply:
I made it optional